index.js

```diff
@@ -2,7 +2,7 @@
 
 // take metadata and remove anything the cli doesnt need to install it.
 const keep = ['versions','name','dist-tags']
-const versionKeep = ['name','version','dependencies','optionalDependencies','devDependencies','bundleDependencies','bundledDependencies','peerDependencies','bin','_hasShrinkwrap','directories','dist','engines','deprecated']
+const versionKeep = ['name','version','dependencies','optionalDependencies','devDependencies','bundleDependencies','bundledDependencies','peerDependencies','bin','_hasShrinkwrap','directories','dist','engines','deprecated','peerDependenciesMeta']
 
 module.exports = function(doc){
```

Comment:
@isaacs is there any contract or guarantee about the `doc` variable being passed in?

This is outside the scope of this pull-request, but I'm just curious if the data coming in is also from our system? Everything looks like it would work correctly. The only thing I can think of in the way of edge cases, is if the keys of `doc.versions` happen to be malformed, or something.

Reply:
Yes, the doc variable being pulled in is a packument, if it's malformed, we have other problems.